daml_dit_if/main/auth_handler.py

```diff
@@ -59,35 +52,23 @@ def get_handler_auth_level(request: "Request") -> 'AuthorizationLevel':
     return getattr(request.match_info.handler, DABL_AUTH_LEVEL, AuthorizationLevel.PUBLIC)
 
 
-def get_ledger_claims(
-        config: 'Configuration', claims: "Mapping[str, Any]") -> "Optional[Mapping[str, Any]]":
-
-    ledger_claims = claims.get('https://daml.com/ledger-api')
-
-    LOG.debug('ledger_claims: %r', ledger_claims)
-
-    if ledger_claims is None:
-        return None
-
-    claimed_ledger_id = ledger_claims.get('ledgerId', 'missing-ledger-id-claim')
-
-    if claimed_ledger_id != config.ledger_id:
-        LOG.debug(f'Ledger ID mismatch in claims: {claimed_ledger_id} != {config.ledger_id}')
-        return None
-
-    return ledger_claims
-
-
-def is_integration_party_ledger_claim(config: "Configuration", ledger_claims: "Mapping[str, Any]") -> bool:
-    act_as_parties = ledger_claims.get('actAs', [])
-    read_as_parties = ledger_claims.get('readAs', [])
-
-    party = config.run_as_party
-
-    if party is None:
-        return False
+def get_token(request: "Request") -> "Optional[str]":
```

Comment:
i realize this was already implemented before, but this function doesn't validate the jwt sig, making it a way to turn a request into some as-of-yet not valid credential.

i believe (and hope) that everywhere this function appears, there is a corresponding jwt sig validation, so as to prevent token tampering from leading to an escalation of scope, but it would be good to verify this

Reply:
I'll rename it to `_unvalidated_get_token`. (There's only a single call site atm.)